### ISSUE TITLE
Patient status can be changed to ‘transfer in progress’ without entering destination

### ISSUE THREAD

**Issue by amruthavijay88** [ohcnetwork/care_fe#760]
Patient status needs to be changed to ‘transfer in progress’ without entering destination

**Comment by vigneshhari** [comment]
do you require a flow of statues? like we can have X status only after Y status and so on ? if so i would require the possible status transfer flows, for eg
PENDING can be moved to APPROVED 
APPROVED can be moved to DESTINATION_APPROVED , DESTINATION_REJECTED and so on for all statuses,

for reference theses are our current statuses : 
    ( "PENDING"),
    ( "ON HOLD"),
    ( "APPROVED"),
    ( "REJECTED"),
    ( "DESTINATION APPROVED"),
    ( "DESTINATION REJECTED"),
    ( "AWAITING TRANSPORTATION"),
    ( "TRANSFER IN PROGRESS"),
    ( "COMPLETED")

**Comment by bodhish** [comment]
@amruthavijay88 @athulmanuel

**Comment by athulmanuel** [comment]
@vigneshhari .. it would be great of each section is only shown the options which each need to perform.

and yes when the card is pushed from one status to other, each criteria has to be filled and completed

once people start using it, we might be able to enter more data into it, per station

**Comment by vigneshhari** [comment]
@amruthavijay88 please update the possible status change flow. 
@athulmanuel once this issue is completed, we can create a feature request to show only the statues that the current person can move to.

**Comment by bodhish** [comment]
@vigneshhari this should be fixed asap.

**Comment by aparnacoronasafe** [comment]
a district admin can still move a shifting request without a destination facility to transfer in progress

**Comment by vigneshhari** [comment]
@aparnacoronasafe the district admin has access to both the shifting facility and the destination facility so they can do it , this is completely expected behaviour

**Comment by gigincg** [comment]
This is resolved
